Sources/ComposableArchitecture/Internal/SwiftUIPreviews.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+@usableFromInline
+var isRunningForPreviews: Bool {
+  #if DEBUG
+  return ProcessInfo.processInfo.environment["XCODE_RUNNING_FOR_PREVIEWS"] == "1"
+  #else
+  return false
+  #endif
+}
```

Comment:
Does it work as a static check that's only performed once?

```suggestion
let isRunningForPreviews = {
  #if DEBUG
  return ProcessInfo.processInfo.environment["XCODE_RUNNING_FOR_PREVIEWS"] == "1"
  #else
  return false
  #endif
}()
```

Reply:
Actually on second thought, how about using `@Dependency(\.context)` and check for `.preview` instead? Then we could maybe even write a test for the behavior.

Reply:
Ok, I'll refactor